### PR TITLE
nghttp2 1.49.0

### PR DESCRIPTION
**Pull request by owine**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/libnghttp2.rb [hunk 1]
```diff
@@ -1,10 +1,9 @@
 class Libnghttp2 < Formula
   desc "HTTP/2 C Library"
   homepage "https://nghttp2.org/"
-  url "https://github.com/nghttp2/nghttp2/releases/download/v1.48.0/nghttp2-1.48.0.tar.gz"
-  mirror "http://fresh-center.net/linux/www/nghttp2-1.48.0.tar.gz"
-  mirror "http://fresh-center.net/linux/www/legacy/nghttp2-1.48.0.tar.gz"
```
Comment (Bo98) [inline Formula/libnghttp2.rb:6]: Legacy mirror should stay for when this gets outdated. Note how http://fresh-center.net/linux/www/nghttp2-1.48.0.tar.gz is now 404 but the legacy mirror works.

**Comment by carlocab** [comment]
`monterey-arm-1` has been uploading bottles for 4+ hours. CC @Bo98

**Comment by carlocab** [comment]
`nghttp2` doesn't seem to break their ABI: https://abi-laboratory.pro/index.php?view=timeline&l=nghttp2

We can probably get away with skipping recursive dependents if we need to run this again.

**Review by carlocab — APPROVED** [review]
`httpd` failures fixed in #108755.

**Pull request closed without merging**